import-service/importfileparser/importfileparser.js

```diff
@@ -20,7 +21,32 @@ module.exports.importFileParser = (event) => {
       s3Stream
         .pipe(csv())
         .on("data", (data) => {
-          console.log(JSON.stringify(data, null, 4));
+          const csvString = JSON.stringify(data);
+          console.log(`csvString: ${csvString}`);
+
+          let products = JSON.parse(csvString);
+          if (!Array.isArray(products)) {
+            products = [products];
+          }
+
+          products.forEach((product) => {
+            sqs.sendMessage(
+              {
+                QueueUrl:
+                  "https://sqs.eu-west-1.amazonaws.com/381077858456/catalogItemsQueue",
```

Comment:
Why not keep this variable in process.env? ("environment" section in serverless.yml?)

Reply:
Yeah, did it late at night and thought a little not about it, I will move it in the future